utils/client_helper.go

```diff
@@ -334,6 +336,67 @@ func HasFIPS(ctx context.Context, client runtimeclient.Client) (bool, error) {
 	return machineConfig.Spec.FIPS, nil
 }
 
+func GetAdditionalTrustBundleFromConfigmap(ctx context.Context, client client.Client, configmapName string) (string, error) {
+	userCaBundleConfigmap := corev1.ConfigMap{}
+	if err := client.Get(ctx, types.NamespacedName{Name: common.ClusterAdditionalTrustBundleName,
+		Namespace: common.OpenshiftConfigNamespace}, &userCaBundleConfigmap); err != nil {
+		if errors.IsNotFound(err) {
+			return "", nil
+		}
+
+		return "", fmt.Errorf("failed to get %s/%s configmap: %w", common.OpenshiftConfigNamespace, common.ClusterAdditionalTrustBundleName, err)
+	}
+
+	if userCaBundleConfigmap.Data == nil {
+		return "", nil
+	}
+
+	if userCaBundleConfigmap.Data[common.CaBundleDataKey] == "" {
+		return "", nil
+	}
+
+	return userCaBundleConfigmap.Data[common.CaBundleDataKey], nil
+}
+
+func GetClusterAdditionalTrustBundleState(ctx context.Context, client client.Client) (bool, string, error) {
+	clusterAdditionalTrustBundle, err := GetAdditionalTrustBundleFromConfigmap(ctx, client, common.ClusterAdditionalTrustBundleName)
```

Comment:
if `configmapName` removed from above suggestion, we may need to remove this one as well
```suggestion
	clusterAdditionalTrustBundle, err := GetAdditionalTrustBundleFromConfigmap(ctx, client)
```